### PR TITLE
Initial commit of poc ui structure

### PR DESCRIPTION
**Pull request by mdrillin**
reorganized code into a structure similar to the apicurio registry.

### CODE REVIEW

File: ui/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (gunnarmorling) [inline ui/LICENSE.txt:1]: That already exists at the repo root, no need to duplicate it.

File: ui/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+
+
+[![License](http://img.shields.io/:license-apache%202.0-brightgreen.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
```
Comment (gunnarmorling) [inline ui/README.md:3]: Same above; could you incorporate the the UI build steps into the root README.

File: ui/pom.xml [hunk 3]
```diff
@@ -0,0 +1,275 @@
+<?xml version="1.0"?>
+<project
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+        xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
```
Comment (gunnarmorling) [inline ui/pom.xml:7]: This one doesn't exist at the given location. Best probably to remove any parent relationship for now and make this one self-contained.

File: ui/pom.xml [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+                <executions>
+                    <!-- Install Node and Yarn -->
+                    <execution>
+                        <id>install node and yarn</id>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>install-node-and-yarn</goal>
+                        </goals>
+                        <configuration>
+                            <nodeVersion>v${node.version}</nodeVersion>
```
Comment (gunnarmorling) [inline ui/pom.xml:51]: Those versions don't exist.

File: ui/pom.xml [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+                                    </excludes>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+    <profiles>
```
Comment (gunnarmorling) [inline ui/pom.xml:111]: Do we need all these profiles actually?

File: ui/packages/ui/src/app/app.tsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import React  from "react";
+import { Page } from "@patternfly/react-core";
+import {BrowserRouter as Router, Route} from 'react-router-dom';
+import AppHeader from "./appHeader";
+import DashboardPage from "./pages/dashboard/dashboard";
```
Comment (indraraj) [inline ui/packages/ui/src/app/app.tsx:5]: Hey @mdrillin , for me the landing page was not coming correctly. Updating import as below helped.

import {DashboardPage} from "./pages";

File: ui/pom.xml (hunk 3 above)
Comment (mdrillin) [reply ui/pom.xml:7]: Yeah, the pom I just copied from the other repo and hadn't looked at it too closely.

File: ui/README.md (hunk 2 above)
Comment (mdrillin) [reply ui/README.md:3]: sure

File: ui/packages/ui/src/app/app.tsx (hunk 6 above)
Comment (mdrillin) [reply ui/packages/ui/src/app/app.tsx:5]: Sure, thanks

File: ui/pom.xml (hunk 3 above)
Comment (gunnarmorling) [reply ui/pom.xml:7]: Ok, are you planning to do this as part of this PR? Or rather would like to have this merged first and follow up on this one separately? Or want me to take a look? Let me know :)

Comment (mdrillin) [reply ui/pom.xml:7]: I was thinking to get this in and then follow up with changes.  Haven't looked at the maven build much yet, just looked at getting a dev ui up for now.  I can do some more cleanup on this today, then maybe just merge this tomorrow and work from there?  We can always re-organize if needed.

Comment (gunnarmorling) [reply ui/pom.xml:7]: Ok, cool. Then I'd say do the changes and clean-up you have in mind and let me know so I can merge it. I'll take a look into the service registry's server side part of the built in the mean time.

Comment (mdrillin) [reply ui/pom.xml]: @gunnarmorling I pushed a few updates per your and @indraraj comments.  I'm not real clear on build / deployment with quarkus, but we can figure out and rearrange if needed as we progress.

**Comment by gunnarmorling** [comment]
Applied. Thanks a lot, @mdrillin 🎉 !

**Pull request merged**